src/EditorFeatures/CSharp/BraceMatching/CSharpEmbeddedLanguageBraceMatcher.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Editor.Implementation.BraceMatching;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.BraceMatching
+{
+    [ExportBraceMatcher(LanguageNames.CSharp)]
+    internal class CSharpEmbeddedLanguageBraceMatcher : AbstractEmbeddedLanguageBraceMatcher
```

Comment:
Only exists because ew cannot export a BraceMatcher against multiple languages.  So all code is actually in the base type.  This type exists just for the export.